LuckySpin/Controllers/SpinnerController.cs

```diff
@@ -13,12 +13,17 @@ public class SpinnerController : Controller
         //TODO: 0) Register the Repository class as a service in Program.cs 
         //TODO: 1) add an instance variable here of type Repository
 
+        Repository repository = new Repository();
```

Comment:
Not quite - you only need to declare the instance variable here
`Repository repository;`